src/BibleWell.App.Android/BibleWell.App.Android.csproj

```diff
@@ -45,6 +45,11 @@
         <PackageReference Include="Xamarin.AndroidX.Core.SplashScreen" />
     </ItemGroup>
 
+    <ItemGroup>
+        <PackageReference Include="Xamarin.Firebase.Messaging" />
+        <GoogleServicesJson Include="google-services.json" />
```

Comment:
Nit: I'd move this to the above item group with app configuration strings and leave this one only for `PackageReference`s.